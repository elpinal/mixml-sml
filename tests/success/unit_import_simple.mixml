[unit
  {unit U : {type a : Type} export (a)}
]
