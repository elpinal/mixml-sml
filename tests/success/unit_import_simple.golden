[{U : [exist (!2 : Type). {a : [= !2]}]-}]+
